### ISSUE TITLE
Render bone blocks (and other new 1.10 blocks)

### ISSUE THREAD

**Issue by fidian** [mapcrafter/mapcrafter#226]
Minecraft 1.10 added bone blocks (# 216) and they act like logs, with a "top".  Would it be possible to add them to the renderer?  Right now they are invisible on rendered maps.

**Comment by m0r13** [comment]
Sure, I will have a look at the bone block and other new 1.10 blocks soon.

**Comment by m0r13** [comment]
Blocks to do:

- [ ] Magma Block
- [ ] Nether Wart Block
- [ ] Red Nether Brick
- [ ] Bone Block
- [ ] Structure Void

**Comment by cliffmeyers** [comment]
+1, this would be great to see.

**Comment by m0r13** [comment]
I got the blocks implemented in the master branch. If you want you can test the feature already when you build Mapcrafter from source (my nightly builds are a bit broken right now, sorry), don't forget to get the proper 1.10 textures.

I have two other little features waiting, and then I'm going to release a new Mapcrafter version.

**Comment by fidian** [comment]
I tried to build the software but it's having trouble with cmake and detecting my g++ supports c++11.  Perhaps it is because my Ubuntu is old-ish?  (14.04)  I gave up and will have to wait for the next release, but I could test a build if you'd want to make one for me, either static or one that works with the likely libraries on my system.

**Comment by m0r13** [comment]
Got the nightly packages fixed: https://mapcrafter.org/downloads/nightly

Just switch to the nightly package repo or download/install a package from there manually.

**Comment by fidian** [comment]
I just noticed that the postinst script downloaded 1.09 instead of 1.10.  This note is just a friendly reminder to up that version before you release the next packaged build of mapcrafter.

**Comment by m0r13** [comment]
Oh thanks a lot for remembering me! I have a special file for that (`MCVERSION`) but seems it's still not used for building the debian packages. I'll have a look at it. Does rendering of the new blocks otherwise work for you too?

**Comment by fidian** [comment]
I started a render of my large map and it could not find the textures.  I ran the command as seen in the invalid ticket and it didn't update my bone_*.png files.  I double checked in the texture folder and the files exist.  So, now I am running a `find` command to delete all image files that were updated today and I'll rerun the map.  The downside is that it takes hours to sort through the image files to find and delete the ones made today.
